tests/integration/test_force_trigger.py

```diff
@@ -501,3 +501,111 @@ async def test_trigger_group_in_flow(
             ('b', '[2]'),
             ('c', '[2]'),
         }
+
+
+async def test_trigger_n0_tasks(
+    flow,
+    scheduler,
+    run,
+    complete,
+    db_select,
+):
+    """It should trigger tasks within their flow if available, else all flows.
+
+    * N=0 tasks already have a flow assigned.
+    * N!=0 tasks do not yet have a flow assigned.
+
+    When we are triggering n!=0 tasks, there is no appropriate flow to run them
+    in (this would involve flow merge prediction), so we default to all active
+    flows as the least objection ate default.
```

Comment:
```suggestion
    flows as the least objectionable default.
```
I'd probably say "a sensible default", or "the only sensible default", but it doesn't really matter in this context.